accounts/abi/bind/bind.go

```diff
@@ -70,16 +70,22 @@ func Bind(types []string, abis []string, bytecodes []string, fsigs []map[string]
 
 		// Extract the call and transact methods; events, struct definitions; and sort them alphabetically
 		var (
-			calls     = make(map[string]*tmplMethod)
-			transacts = make(map[string]*tmplMethod)
-			events    = make(map[string]*tmplEvent)
-			structs   = make(map[string]*tmplStruct)
+			calls       = make(map[string]*tmplMethod)
+			transacts   = make(map[string]*tmplMethod)
+			events      = make(map[string]*tmplEvent)
+			structs     = make(map[string]*tmplStruct)
+			identifiers = make(map[string]bool)
```

Comment:
nitpick: to save some memory you could declare it as `map[string]struct{}.

Reply:
yeah, but `make(map[string]bool)` is simpler :P. I don't need to check like `_, exist := map[id]; exist`.